lib/explorer/series.ex

```diff
@@ -2528,6 +2528,90 @@ defmodule Explorer.Series do
     basic_numeric_operation(:covariance, left, right, [ddof])
   end
 
+  @doc """
+  Returns if all the values in a boolean series are true.
+
+  ## Supported dtypes
+
+    * `:boolean`
+
+  ## Examples
+
+      iex> s = Series.from_list([true, true, true])
+      iex> Series.all?(s)
+      true
+
+      iex> s = Series.from_list([true, false, true])
+      iex> Series.all?(s)
+      false
+
+      iex> s = Series.from_list([1, 2, 3])
+      iex> Series.all?(s)
+      ** (ArgumentError) Explorer.Series.all?/1 not implemented for dtype :integer. Valid dtypes are [:boolean]
+
+  An empty series will always return true:
+
+      iex> s = Series.from_list([], dtype: :boolean)
+      iex> Series.all?(s)
+      true
+
+  Nil values are ignored:
```

Comment:
```suggestion
  Opposite to Elixir but similar to databases, `nil` values are ignored:
```

Reply:
I thought PostgreSQL always followed Kleene logic for boolean operations, but apparently, that's just true for `AND`, `OR`, etc. The `bool_and` and `bool_or` aggregations just ignore `NULL`s.

TIL!